### DISCUSSION TITLE
Newbie here!!

### DISCUSSION THREAD

**Discussion by saiirrxx** [community/community#39577] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

HI, I'm new here on github, may I ask how can I connect my account on canva? badly need for my education. tyia!!! and please enlighten me how to explore github and site or orgs to connect that help my studies.

**Comment by DSDmark** [comment]
Hi @saiirrxx,
   
> Nice to meet you and welcome to this great community, we are here to help each other and try to make open source better. This is the discussions section if you want to discuss any. you can ask here, we will try to help.

Or why any chance if you are curious about, 

> what is GitHub, who is GitHub then I highly suggest you check this out [**What is GitHub?**](https://youtu.be/pBy1zgt0XPc).  its a video. so, just ***sit back and relax***

So, About the **Canva**,

> The Canva benefit is no longer offered in the GitHub Student Developer Pack.

Here you can read about more:- [**discussion about Canva**](https://github.com/orgs/community/discussions/28136#discussioncomment-3319262)
